tutorials/2d/custom_drawing_in_2d.rst

```diff
@@ -69,12 +69,14 @@ redrawn if modified:
 
     extends Node2D
 
-    export (Texture) var texture setget _set_texture
+    @export var texture : Texture :
+        set(value):
+            texture = value
+            _set_texture(value)
 
     func _set_texture(value):
         # If the texture variable is modified externally,
         # this callback is called.
```

Comment:
This comment can be misleading, since in 4.0 setters and getters are always called (both externally and internally).